Earthfile

```diff
@@ -0,0 +1,83 @@
+
```

Comment:
I found out this blank line is important when my editor automatically stripped it and I got the following error:

```
% sudo earth -P --build-arg ELIXIR=1.11.1 --build-arg OTP=21.3.8.18 +integration-test

           buildkitd | Found buildkit daemon as docker container (earthly-buildkitd)
line 1:0 no viable alternative at input 'all:'
Error: build target: build main: bkClient.Build: failed to solve: rpc error: code = Unknown desc = syntax error: line 1:0 no viable alternative at input 'all:'
```

It looks like you've already opened an issue on it https://github.com/earthly/earthly/issues/538, but I wanted to call that out in case anyone else is trying this out.